docs/index.asciidoc

```diff
@@ -2,9 +2,13 @@
 :server-branch: 6.5
 include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
 
-:type-string: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type[`<string>`]
-:type-boolean: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Boolean_type[`<boolean>`]
-:type-object: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object[`<Object>`]
+:type-string: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type[<string>]
```

Comment:
🤔 how come this works? I would have thought that we needed to add the inline code-block backtick (`), but even without them it all seems to render corrctly with a monospace font when used.

Reply:
![magic](https://user-images.githubusercontent.com/5618806/55817294-12080880-5aa9-11e9-9641-18f7a76fb62b.gif)

You can't describe text format within the attribute, you have to do it where the attribute is referenced. It works because of the plus signs we use to reference the attributes: `+{type-object}+`. The plus signs do the same as the backticks, but they allow attribute substitution.

Reply:
Ohhhh TIL! 🤓